src/utils/mtev_log.c

```diff
@@ -3191,6 +3191,26 @@ mtev_log_init_globals(void) {
   }
 }
 
+void
+mtev_log_hexdump(mtev_log_stream_t* ls, const void * addr, const unsigned int len) {
```

Comment:
it is more common for these types of routings to take a `size_t` point intead of an `unsigned int` pointer.